qiskit/transpiler/passes/layout/vf2_utils.py

```diff
@@ -12,31 +12,18 @@
 
 """This module contains common utils for vf2 layout passes."""
 
+__all__ = ["ErrorMap"]
+
 from collections import defaultdict
-import random
 
-import numpy as np
 from rustworkx import PyDiGraph, PyGraph, connected_components, weakly_connected_components
 
+from qiskit._accelerate.error_map import ErrorMap
 from qiskit.circuit import ForLoopOp
 from qiskit.converters import circuit_to_dag
-from qiskit.transpiler.target import Target
-from qiskit._accelerate import vf2_layout
-from qiskit._accelerate.nlayout import NLayout
-from qiskit._accelerate.error_map import ErrorMap
-
-
-def allocate_idle_qubits(dag, target, layout):
-    """Allocate the idle virtual qubits in the input DAG to arbitrary physical qubits."""
-    # Extend with arbitrary decisions for idle qubits.
-    used_physical = set(layout.get_physical_bits())
-    unused_physicals = (q for q in range(target.num_qubits) if q not in used_physical)
-    for bit in dag.qubits:
-        if bit not in layout:
-            layout[bit] = next(unused_physicals)
-    return layout
 
 
+# This function is (unfortunately) used by non-VF2 places in Qiskit.
```

Comment:
Only by a test in `test_star_prerouting`, right? If we really need this function there, we could just move it there, and get rid of it here.

Reply:
I think it was used more at the time I wrote the commit, but since it's no longer used, I removed it in 1859960967.  I just rewrote the simple `StarPreroutingTest` to extract the edges without all the extra baggage.